pki.spec

```diff
@@ -149,7 +153,8 @@ BuildRequires:    make
 BuildRequires:    cmake >= 3.0.2
 BuildRequires:    gcc-c++
 BuildRequires:    zip
-BuildRequires:    java-1.8.0-openjdk-devel
+BuildRequires:    java-devel >= 1:1.8.0
```

Comment:
Should we depend on JDK 11 on F33+ (in case JDK 8 is dropped from future F33+)?

Reply:
Heh, sorry. I had that change sitting locally but hadn't yet pushed. Yes, I've made that change explicitly. :)